mscore/prefsdialog.cpp

```diff
@@ -250,7 +247,248 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
+                  new BoolPreferenceItem(PREF_IMPORT_MUSICXML_IMPORTLAYOUT, importLayout),
+            #ifdef AVSOMR
+                  new BoolPreferenceItem(PREF_IMPORT_AVSOMR_USELOCAL, useLocalAvsOmr, [&](){ updateUseLocalAvsOmr(); }),
+            #endif
+                  new BoolPreferenceItem(PREF_IO_MIDI_ADVANCEONRELEASE, advanceOnRelease),
+                  new BoolPreferenceItem(PREF_IO_MIDI_ENABLEINPUT, enableMidiInput),
+                  new BoolPreferenceItem(PREF_IO_MIDI_EXPANDREPEATS, expandRepeats),
+                  new BoolPreferenceItem(PREF_EXPORT_AUDIO_NORMALIZE, normalize),
+                  new BoolPreferenceItem(PREF_IO_MIDI_EXPORTRPNS, exportRPNs),
+                  new IntPreferenceItem(PREF_IO_MIDI_REALTIMEDELAY, realtimeDelay),
+                  new BoolPreferenceItem(PREF_IO_MIDI_USEREMOTECONTROL, rcGroup),
+                  new IntPreferenceItem(PREF_IO_OSC_PORTNUMBER, oscPort),
+                  new BoolPreferenceItem(PREF_IO_OSC_USEREMOTECONTROL, oscServer),
+                  new BoolPreferenceItem(PREF_SCORE_CHORD_PLAYONADDNOTE, playChordOnAddNote),
+                  new BoolPreferenceItem(PREF_SCORE_HARMONY_PLAY, playHarmony),
+                  new BoolPreferenceItem(PREF_SCORE_HARMONY_PLAY_ONEDIT, playHarmonyOnEdit),
+                  new IntPreferenceItem(PREF_SCORE_NOTE_DEFAULTPLAYDURATION, defaultPlayDuration),
+                  new BoolPreferenceItem(PREF_SCORE_NOTE_PLAYONCLICK, playNotes),
+                  new BoolPreferenceItem(PREF_UI_APP_STARTUP_CHECKUPDATE, checkUpdateStartup),
+                  new BoolPreferenceItem(PREF_UI_APP_STARTUP_SHOWNAVIGATOR, navigatorShow),
+                  new BoolPreferenceItem(PREF_UI_APP_STARTUP_SHOWPLAYPANEL, playPanelShow),
+                  new BoolPreferenceItem(PREF_UI_APP_STARTUP_SHOWSPLASHSCREEN, showSplashScreen),
+                  new BoolPreferenceItem(PREF_UI_APP_STARTUP_SHOWSTARTCENTER, showStartcenter),
+                  new BoolPreferenceItem(PREF_UI_APP_STARTUP_SHOWTOURS, showTours),
+                  new BoolPreferenceItem(PREF_APP_TELEMETRY_ALLOWED, collectTelemetry),
+                  new BoolPreferenceItem(PREF_IO_JACK_TIMEBASEMASTER, becomeTimebaseMaster),
+                  new BoolPreferenceItem(PREF_IO_JACK_REMEMBERLASTCONNECTIONS, rememberLastMidiConnections),
+                  new BoolPreferenceItem(PREF_SCORE_NOTE_WARNPITCHRANGE, warnPitchRange),
+                  new StringPreferenceItem(PREF_IMPORT_OVERTURE_CHARSET, importCharsetListOve, nullptr, [&](){ updateCharsetListOve(); }),
+                  new StringPreferenceItem(PREF_IMPORT_GUITARPRO_CHARSET, importCharsetListGP, nullptr, [&](){ updateCharsetListGP(); }),
+                  new DoublePreferenceItem(PREF_SCORE_MAGNIFICATION, scale,
+                                          [&](){ preferences.setPreference(PREF_SCORE_MAGNIFICATION, scale->value()/100.0); },      // apply function
```

Comment:
spaces around `/`:
`preferences.setPreference(PREF_SCORE_MAGNIFICATION, scale->value() / 100.0); `
and not
`preferences.setPreference(PREF_SCORE_MAGNIFICATION, scale->value()/100.0); `